### PR TITLE
updated requirements.txt

### PR DESCRIPTION
**Pull request by ADMoreau**
added requests and scikit-learn to requirements.txt

### CODE REVIEW

**Comment by gmuraru** [comment]
Aren't those needed only for notebooks?

**Comment by ADMoreau** [comment]
The scikit learn is for the create-sandbox so I assume that is only for notebooks. I'm getting the requests requirement for FixedPrecisionTensor. I'm not sure if this is only for notebooks. Got them both while doing some debugging -> not running full code, so maybe

**Comment by ADMoreau** [comment]
`Traceback (most recent call last):
  File "/home/austin/PycharmProjects/untitled7/test.py", line 2, in <module>
    from syft.frameworks.torch.tensors.interpreters.precision import FixedPrecisionTensor
  File "/home/austin/.virtualenvs/untitled7/lib/python3.6/site-packages/syft/__init__.py", line 44, in <module>
    from syft.grid.public_grid import PublicGridNetwork
  File "/home/austin/.virtualenvs/untitled7/lib/python3.6/site-packages/syft/grid/public_grid.py", line 2, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'`

**Comment by ADMoreau** [comment]
Traceback (most recent call last):
  File "/home/austin/PycharmProjects/untitled7/test.py", line 2, in <module>
    from syft.frameworks.torch.tensors.interpreters.precision import FixedPrecisionTensor
  File "/home/austin/.virtualenvs/untitled7/lib/python3.6/site-packages/syft/__init__.py", line 44, in <module>
    from syft.grid.public_grid import PublicGridNetwork
  File "/home/austin/.virtualenvs/untitled7/lib/python3.6/site-packages/syft/grid/public_grid.py", line 2, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'

**Comment by gmuraru** [comment]
In that case, I think we should add only the requirements. Ping @IonesioJunior since he worked on the ```public_grid.py```, maybe you have further feedback :)

**Comment by IonesioJunior** [comment]
Yep! Thank you to fix this! I really forgot to update the requirements.txt file.

**Comment by gmuraru** [comment]
@ADMoreau could you fix the conflict and then we are good to merge :)

**Comment by ADMoreau** [comment]
Done on my end, glad to help

File: pip-dep/requirements.txt [hunk 1]
```diff
@@ -3,13 +3,15 @@ flask_socketio>=3.3.2
...
-Pillow<7
 scipy>=1.4.1
 syft-proto>=0.1.1.a1.post2
 tblib>=1.4.0
 torch==1.4
 torchvision==0.5.0
+phe>=1.4.0
+Pillow<7
+requests==2.22.0
+scikit-learn==0.20.4
```
Comment (gmuraru) [inline pip-dep/requirements.txt]: I think this could be removed since we also have another requirement file needed for the notebooks :)

**Comment by ADMoreau** [comment]
Sounds good to me!

**Comment by ADMoreau** [comment]
Sorry I wasn't paying attention and had to reset the pull request

**Comment by gmuraru** [comment]
Thank you for the PR. Once Travis passes I will merge it :)

**Pull request merged**